numba/tests/test_typedlist.py

```diff
@@ -689,6 +826,312 @@ def foo():
         self.assertEqual(0, got[0])
 
 
+class TestDisableReflectedListBase(MemoryLeakMixin, TestCase):
+
+    def _njit_both(self, func):
+        return (njit(func, _disable_reflected_list=p) for p in (True, False))
+
+    def _check(self,
+               foo_true_expected, foo_true_received, foo_true_type,
+               foo_false_expected, foo_false_received, foo_false_type,
+               ):
+        self.assertEqual(foo_true_expected, foo_true_received)
+        self.assertEqual(foo_false_expected, foo_false_received)
+        self.assertEqual(foo_true_type, type(foo_true_received))
+        self.assertEqual(foo_false_type, type(foo_false_received))
```

Comment:
`self.assertPreciseEqual()` checks for type too. Is this method not working with `typed.List` at present?

Reply:
no, when I use `assertPreciseEqual` I get an error about `'unknown' != 'sequence'` presume the typed-list still needs to identify itself as `sequence`?

Reply:
quite possibly, `assertPreciseEquals` predates typed list by about 5 years.

Reply:
`assertPreciseEqual` could be updated to include `numba.typed.List` in the `sequence` family.